### PR TITLE
Add "Providing your own dummy data"

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: docs/study-def-expectations.md [hunk 1]
```diff
@@ -98,3 +100,25 @@ The following options are currently available for dummy data (note numeric value
...
+  dummy_data_file: test-data/dummy-data.csv
+  outputs:
+    highly_sensitive:
+      cohort: output/input.csv
+
+```
+
+The dummy data file must be commmitted to the repo.
+You should generate the dummy data using a script, and commit the script to the repo.
+You must not use real clinical data as dummy data!
```
Comment (StevenMaude) [inline docs/study-def-expectations.md:124]: Maybe worth adding `**`s around this last point, or put in a warning box via `!!! warning`? Not obligatory, but it seems worth emphasising.

File: docs/study-def-expectations.md [hunk 2]
```diff
@@ -98,3 +100,25 @@ The following options are currently available for dummy data (note numeric value
...
+generate_cohort:
+  run: cohortextractor:latest generate_cohort
+  dummy_data_file: test-data/dummy-data.csv
+  outputs:
+    highly_sensitive:
+      cohort: output/input.csv
+
+```
+
+The dummy data file must be commmitted to the repo.
```
Comment (StevenMaude) [inline docs/study-def-expectations.md:122]: nit: one too many "m" here in "committed".

File: docs/news.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 
 This page lists significant improvements to the platform since June 2021, with the most recent at the top. We suggest you check it regularly.
 
+* 2021-07-21: Added [support for users to provide their own dummy data](/study-def-expectations#providing-your-own-dummy-data)
```
Comment (StevenMaude) [inline docs/news.md:4]: Well done for including this here :+1:

File: docs/study-def-expectations.md (hunk 2 above)
Comment (inglesp) [reply docs/study-def-expectations.md:122]: That's nnot a nnit!

**Review by StevenMaude — APPROVED** [review]
A couple of minor changes I've suggested; at minimum, please fix the typo :slightly_smiling_face:

(Indirectly related to this PR, my view is that I do not much like the term "dummy data". But that's out of scope here as the documentation change here is reflecting current usage.)

**Pull request merged**